CMR-Metadata-Script/cmrmetadata.py

```diff
@@ -0,0 +1,56 @@
+import csv
+from datetime import datetime
+from datetime import timezone
+from pip._vendor import requests
+import uuid
+from config import BaseConfig
+
+base_url = BaseConfig.CMR_BASE_URL
+query_url = base_url + BaseConfig.GRANULES_URL
+
+filename = BaseConfig.OUTPUT_FILE_NAME
+extension = BaseConfig.OUTPUT_FILE_EXTENSION
+
+outfile = "cmrmetadata.csv"
+
+concept_id = BaseConfig.CONCEPT_ID
+
+page_num = BaseConfig.PAGE_NUM
+page_size = BaseConfig.PAGE_SIZE
+
+s3_bucket_name = BaseConfig.S3_BUCKET_NAME
+
+start_date = BaseConfig.START_DATE
+end_date = BaseConfig.END_DATE
+
+date_range = start_date + ',' + end_date
+
+bounding_box = BaseConfig.BOUNDING_BOX
+
+page_num_limit = BaseConfig.PAGE_NUM_LIMIT
+
+with open(outfile, "w", newline= "") as f:
+    writer = csv.writer(f,delimiter=",")
+    writer.writerow(["Id", "Name", "ContentLength", "IngestionDate", "ContentDate:Start", "ContentDate:End", "Checksum:Algorithm", "Checksum:Value", "IsSealed", "SourceStorageId", "DestStorageId"])
+    while (page_num <= page_num_limit):
+        formated_url = query_url.format(concept_id, page_num, page_size, date_range, bounding_box)
+        print(formated_url)
+        resp = requests.get(query_url.format(concept_id,page_num,page_size, date_range, bounding_box)).json()
+        entries = resp["feed"]["entry"]
+        for i,granule in enumerate(entries):
+            for link in granule["links"]:
+                if (link["href"].startswith(s3_bucket_name)):
+                    Id = uuid.uuid1()
+                    Name = link["href"].split(s3_bucket_name)[1]
+                    IngestionDate = datetime.fromisoformat(granule["time_start"][:-1]).astimezone(timezone.utc).strftime('%Y-%m-%d %H:%M:%S')
+                    ContentDateStart = datetime.fromisoformat(granule["time_start"][:-1]).astimezone(timezone.utc).strftime('%Y-%m-%d %H:%M:%S')
+                    ContentDateEnd = datetime.fromisoformat(granule["time_end"][:-1]).astimezone(timezone.utc).strftime('%Y-%m-%d %H:%M:%S')
+                    ContentLength = 100
+                    ChecksumAlgorithm = 'MD5'
```

Comment:
is this checksum value always the same?

Reply:
Please use SHA-256 or better rather than MD5.

Reply:
> is this checksum value always the same?

@NISH1001, In CMR we don't have an MD5 hash for files, so hard coding MD5 for now. 
If in the future CMR includes MD5, we can accommodate it in the script.